mne/chpi.py

```diff
@@ -913,6 +952,44 @@ def _unit_quat_constraint(x):
     return 1 - (x * x).sum()
 
 
+@verbose
+def compute_chpi_snr(raw, t_step_min=0.01, t_window='auto', ext_order=1,
+                     tmin=0, tmax=None, verbose=None):
+    """Compute time-varying estimates of cHPI SNR.
+
+    Parameters
+    ----------
+    raw : instance of Raw
+        Raw data with cHPI information.
+    t_step_min : float
+        Minimum time step to use. If correlations are sufficiently high,
+        t_step_max will be used.
```

Comment:
there is no t_step_max?

Reply:
I meant to ask about that...  it's a copy-paste from the `compute_chpi_amplitudes` docstring, which also doesn't have a `t_step_max`...  I was hoping you would know why